mobly/controllers/attenuator_lib/telnet_scpi_client.py

```diff
@@ -70,7 +70,7 @@ def cmd(self, cmd_str, wait_ret=True):
       return None
 
     match_idx, match_val, ret_text = self._tn.expect(
-        [_ascii_string("\S+" + self.rx_cmd_separator)], 1)
+        [_ascii_string(r'\S+' + self.rx_cmd_separator)], 1)
```

Comment:
Fix according to Flake8 rule [W605](https://www.flake8rules.com/rules/W605.html)
> W605 invalid escape sequence '\S'